api/client/trust.go

```diff
@@ -37,7 +37,10 @@ import (
 	"github.com/docker/notary/tuf/data"
 )
 
-var untrusted bool
+var (
+	releasesRole = data.CanonicalTargetsRole + "/releases"
```

Comment:
`filepath.Join()` was definitely wrong, using `path.Join()` instead of concatenation, is just a nit pick, so, all is well :+1: